packages/pluggableWidgets/datagrid-number-filter-web/src/components/__tests__/__snapshots__/DatagridNumberFilter.spec.tsx.snap

```diff
@@ -1,57 +1,63 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Number Filter with single instance with multiple attributes renders correctly 1`] = `
-<div
-  class="filter-container filter-custom-class"
-  data-focusindex="0"
->
+<DocumentFragment>
   <div
-    class="filter-selector"
+    class="filter-container filter-custom-class"
+    data-focusindex="0"
   >
     <div
-      class="filter-selector-content"
+      class="filter-selector"
     >
-      <button
-        aria-controls="NumberFilter1-filter-selectors"
-        aria-expanded="false"
-        aria-haspopup="true"
-        class="btn btn-default filter-selector-button button-icon equal"
+      <div
+        class="filter-selector-content"
       >
-         
-      </button>
+        <button
+          aria-controls="NumberFilter1-filter-selectors"
+          aria-expanded="false"
+          aria-haspopup="true"
+          class="btn btn-default filter-selector-button button-icon equal"
+        >
+           
+        </button>
+      </div>
     </div>
+    <input
+      class="form-control filter-input"
+      type="number"
+      value=""
```

Comment:
This will now always have an empty string value, is that fine? (The others didn't have such a change)